### PR TITLE
Add common and base test scripts for nightly AddressSanitizer testing

### PR DESCRIPTION
**Pull request by e-kayrakli**
The common file is very simple and sets the environment as described in
https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/Sanitizers.rst

The actual test script starts with standard config by sourcing `common.bash`
and fires off nightly with 6 tasks per node as we do in other chapcs
correctness tests.

### CODE REVIEW

File: util/cron/common-asan.bash [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Configure environment for testing AddressSanitizer.
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+export CHPL_MEM=cstdlib
+export CHPL_TASKS=fifo
+export CHPL_SANITIZE_EXE=address
```
Comment (ronawho) [inline util/cron/common-asan.bash:9]: Any reason to not sanitize the compiler too (`export CHPL_SANITIZE=address`)?

Comment (e-kayrakli) [reply util/cron/common-asan.bash:9]: I just remembered that it was causing compiler performance slowdowns, but maybe I was wrong?

Comment (ronawho) [reply util/cron/common-asan.bash:9]: The early implementation you were using was throwing `-g -O1`, which I think was the main cause of overhead. I'd probably start with compiler sanitizing enabled, and we can evaluate if it's too much slowdown later.

Comment (e-kayrakli) [reply util/cron/common-asan.bash:9]: Ah, right. Sounds good.

**Pull request merged**